src/language-client.ts

```diff
@@ -468,7 +468,16 @@ export class LanguageClient implements Disposable {
     }
   }
 
-  public async sendWillSaveWaitUntil (document: TextDocument, reason: TextDocumentSaveReason, token?: rpc.CancellationToken): Promise<TextEdit[] | null> {
+  public async sendWillSaveNotification (uri: string, reason: TextDocumentSaveReason): Promise<void> {
+    const document = this.currentDocuments.get(uri)
+    if (document == null) {
+      throw new Error('sendWillSaveNotification called for an unknown document')
```

Comment:
You shouldn't probably throw an error because currentDocuments is empty if the server doesn't support didOpen/didClose I think

Reply:
I think

Reply:
[fix: ignore if document doesn't exist](https://github.com/CodinGame/languageserver-mutualized/pull/10/commits/8eb5c65fc90587bb20f98315c52fd88e58aa174c)

Reply:
also I removed eslint warnings: [fix(eslint): remove warnings](https://github.com/CodinGame/languageserver-mutualized/pull/10/commits/d3536857e9db00c11696bf01b3f903d441658755)